CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ##### Enhancements
 
+* [8785](https://github.com/grafana/loki/pull/8785) **dannykopping**: Ruler: remote rule evaluation hardening.
```

Comment:
I don't think this needs an extra changelog entry. It's part of the "remote rule evaluation" feature.

Reply:
That's fair 👍 added it reflexively